xonsh/parsers/base.py

```diff
@@ -26,6 +26,7 @@
 RE_STRINGPREFIX = LazyObject(
     lambda: re.compile(StringPrefix), globals(), "RE_STRINGPREFIX"
 )
+RE_FSTR_ENVVAR = LazyObject(lambda: re.compile("\{\s*\$(\w+)"), globals(), "RE_FSTR_ENVVAR")
```

Comment:
`black` wants this line to be shorter (sorry, i need to add that check to CI)

Reply:
done! Thanks

Reply:
I meant to run black, but forgot :(